site/src/components/ParameterInput/ParameterInput.tsx

```diff
@@ -3,10 +3,26 @@ import Radio from "@material-ui/core/Radio"
 import RadioGroup from "@material-ui/core/RadioGroup"
 import { makeStyles } from "@material-ui/core/styles"
 import TextField from "@material-ui/core/TextField"
+import { Stack } from "components/Stack/Stack"
 import { FC } from "react"
 import { ParameterSchema } from "../../api/typesGenerated"
 import { MONOSPACE_FONT_FAMILY } from "../../theme/constants"
 
+const isBoolean = (schema: ParameterSchema) => {
+  return schema.validation_value_type === "bool"
```

Comment:
would be nice to have an enum for this

Reply:
That is true, but I would wait until we use it more often or with different values.